### PR TITLE
Added file for PowerShell support

### PR DESCRIPTION
**Pull request by digitalguy99**
I've written a PowerShell(.ps1) equivalent for ghci-color.sh. The program works despite containing some bugs as stated below:

- Prompt occasionally adds an unwanted indentation after a command is executed.
- Error message doesn't show after the execution of an unwanted command. (fixed)

Suggestions are welcome.

### CODE REVIEW

**Comment by rhysd** [comment]
Is this ready for review?

**Comment by digitalguy99** [comment]
@rhysd yes, it is.

File: README.md [hunk 1]
```diff
@@ -11,17 +11,41 @@ this script colorizes ghci output.
...
 
-put ghci-color script in $PATH directory and use ghci-color command instead of ghci.
+```
+mv ghci-color <$PATH directory>
+```
+<br>
 
+Run `ghci-color` instead of ghci.
+<br>
+<br>
```
Comment (rhysd) [inline README.md:48]: Can you remove all `&nbsp` and `<br>` since they makes readme less readable.

**Review by rhysd — CHANGES_REQUESTED** [review]

**Review by rhysd — APPROVED** [review]
Thank you! Looks good.

**Pull request merged**